var/spack/repos/builtin/packages/meson/package.py

```diff
@@ -19,6 +19,9 @@ class Meson(PythonPackage):
 
     license("Apache-2.0")
 
+    version("1.7.0", sha256="a6ca46e2a11a0278bb6492ecd4e0520ff441b164ebfdef1e012b11beb848d26e")
+    version("1.6.1", sha256="4889795777b536ea1a351982f3ef7c7b06a786ccb47036daba63cc5757c59edb")
+    version("1.5.2", sha256="fb41882bef26ffc02647d9978cba502a4accdf2e94c0a6dc9cc498dd7463381e")
     version("1.5.1", sha256="55f6acd5bf72c14d4aa5a781993633f84a1d117bdf2c2057735902ced9b81390")
```

Comment:
Please deprecate 1.5.1 if you're packaging a newer bugfix release in the same major.minor version.